dbms/src/Interpreters/TraceLog.cpp

```diff
@@ -23,10 +24,11 @@ Block TraceLogElement::createBlock()
         {std::make_shared<DataTypeDate>(),                                    "event_date"},
         {std::make_shared<DataTypeDateTime>(),                                "event_time"},
         {std::make_shared<DataTypeUInt32>(),                                  "revision"},
-        {std::make_shared<TimerDataType>(timer_values),                       "timer_type"},
+        {std::make_shared<TraceDataType>(trace_values),                       "trace_type"},
         {std::make_shared<DataTypeUInt32>(),                                  "thread_number"},
         {std::make_shared<DataTypeString>(),                                  "query_id"},
-        {std::make_shared<DataTypeArray>(std::make_shared<DataTypeUInt64>()), "trace"}
+        {std::make_shared<DataTypeArray>(std::make_shared<DataTypeUInt64>()), "trace"},
+        {std::make_shared<DataTypeInt64>(),                                   "size"},
```

Comment:
I'd suggest `allocated_bytes`.

Reply:
It's Ok. Because what else we will add to the trace_log.